web/src/show.php

```diff
@@ -60,6 +60,15 @@
             'id' => $property['person_id']
         );
     }
+    $lecturefields = array(
+        'person_id' => 'Student',
+        'semester_abs' => 'Semester',
+        'semester_rel' => 'Vorname',
```

Comment:
Should be "ordinal", not "Vorname". It's the ordinal no. of the semester relative to the student's start date, while the other is the absolute value with year and summer/winter descriptor.